pymc_bart/pgbart.py

```diff
@@ -116,20 +168,15 @@ def __init__(
 
         self.tune = True
 
-        if batch == "auto":
-            batch = max(1, int(self.m * 0.1))
-            self.batch = (batch, batch)
-        else:
-            if isinstance(batch, (tuple, list)):
-                self.batch = batch
-            else:
-                self.batch = (batch, batch)
+        if batch == (0.1, 0.1):
+            batch_component = max(1, int(self.m * 0.1))
+            self.batch = (batch_component, batch_component)
```

Comment:
This should be something like this. 


```suggestion
    batch_0 = max(1, int(self.m * batch[0]))
    batch_1 = max(1, int(self.m * batch[1]))
    self.batch = ( batch_0,  batch_1)
```